src/envo/envo-edit.owl

```diff
@@ -48367,6 +48456,12 @@ SubClassOf(<http://purl.obolibrary.org/obo/OBI_0000967> <http://purl.obolibrary.
 
 SubClassOf(<http://purl.obolibrary.org/obo/OBI_0000968> <http://purl.obolibrary.org/obo/ENVO_00003074>)
 
+# Class: <http://purl.obolibrary.org/obo/PATO_0001486> (chronological age)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/PATO_0001486> "An age quality that exists by virtue of the time (years and months) that the bearer has existed.")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/PATO_0001486> "chronological age")
+SubClassOf(<http://purl.obolibrary.org/obo/PATO_0001486> <http://purl.obolibrary.org/obo/PATO_0000011>)
+
```

Comment:
Was this added manually or via a process like https://github.com/EnvironmentOntology/envo/wiki/Import-terms-from-other-ontologies?

Reply:
This was added manually using Protege. I can look through the process shortly though and take another run at it.

Reply:
It takes some docker magic, but isn't that hard if you have the memory (around 16 GB for the container is usually enough, processing the CHEBI and NCBITaxon imports hog memory)

Reply:
Added PATO chronologic age import as issue #1587